source/features/git-checkout-pr.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import ClippyIcon from 'octicon/clippy.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {getCurrentBranch} from '../github-helpers';
+
+function handleMenuOpening({delegateTarget: dropdown}: delegate.Event): void {
+	dropdown.classList.add('rgh-git-checkout'); // Mark this as processed
+	const [, user, repository] = select<HTMLAnchorElement>('.commit-ref.head-ref a')!.pathname.split('/', 3);
+	const isLocalPr = select('.user-select-contain.head-ref a')!.childElementCount === 1;
+	const tabContainer = select('[action="/users/checkout-preference"]', dropdown)!.closest<HTMLElement>('tab-container')!;
+	tabContainer.style.width = 'fit-content';
+	select('.UnderlineNav-body', tabContainer)!.append(
+		<button
+			name="type"
+			type="button"
+			role="tab"
+			aria-selected="false"
+			className="UnderlineNav-item flex-1 btn-link"
+		>
+			Git Checkout
+		</button>
+	);
+
+	tabContainer.append(
+		<div hidden role="tabpanel" className="p-3">
+			<p className="text-gray text-small">
+				Run in your project repository
+			</p>
+			<div className="copyable-terminal">
+				<div className="copyable-terminal-button">
+					<clipboard-copy
+						className="btn btn-sm zeroclipboard-button"
+						role="button"
+						for="rgh-checkout-pr"
+						aria-label="Copy to clipboard"
+						data-copy-feedback="Copied!"
+					>
+						<ClippyIcon/>
+					</clipboard-copy>
+				</div>
+				<pre
+					id="rgh-checkout-pr"
+					className="copyable-terminal-content"
+				>
+					<span className="user-select-contain">
+						{isLocalPr ? '' : `git remote add ${user} ${location.origin}/${user}/${repository}.git\n`}
+						git switch {user}/{getCurrentBranch()}
```

Comment:
This command doesn't work for me:

```
❯ git remote add yakov116 https://github.com/yakov116/refined-github.git
❯ git switch yakov116/check-it-out
fatal: invalid reference: yakov116/check-it-out
```

`git checkout user/branch` doesn't work either

```
❯ git checkout yakov116/check-it-out
error: pathspec 'yakov116/check-it-out' did not match any file(s) known to git
```

The remote worked:
```
❯ git remote
fregante
origin
yakov116
```

Reply:
It needs a fetch after the remote is added: `git fetch yakov116`